Dockerfile

```diff
@@ -1,6 +1,8 @@
-# This image is based on the official sonar-scanner image
+ # This image is based on the official sonar-scanner image
 FROM sonarsource/sonar-scanner-cli:4.4
 
 LABEL maintainer="LEQUAL"
 
 # TODO: install all tools available in Docker-CAT
+
+MAINTAINER Benoît and Thomas
```

Comment:
Codacy found an issue: [MAINTAINER is deprecated](https://app.codacy.com/gh/lequal/sonar-scanner/pullRequest?prid=5967771)